### PR TITLE
feat: [FFM-7750]: swapped variation name and input field

### PR DESCRIPTION
**Pull request by crystal-fong**
### Summary

Swapping the Variation name/Icon with the percentage rollout input

In Feature Flags Details
image

In Target Group Details

image

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/constants/index.ts [hunk 1]
```diff
@@ -96,6 +97,21 @@ export const CFVariationColors = [
...
+  Color.GREEN_600,
+  Color.GREY_200,
+  Color.PRIMARY_5,
+  Color.RED_400,
+  Color.YELLOW_900,
+  Color.GREY_400,
+  Color.PURPLE_400,
+  Color.ORANGE_500,
+  Color.MAGENTA_50
+]
```
Comment (crystal-fong) [inline src/modules/75-cf/constants/index.ts:113]: had to introduce this array as `iconProps = {{ color:  }}` won't accept hex values

File: src/modules/75-cf/constants/index.ts [hunk 2]
```diff
@@ -96,6 +97,21 @@ export const CFVariationColors = [
   '#ed61b5'
 ]
 
+export const CFVariationColorsColorRef = [
```
Comment (knagurski) [inline src/modules/75-cf/constants/index.ts:100]: Can we drop the `CF` part?

Comment (crystal-fong) [reply src/modules/75-cf/constants/index.ts:100]: I wanted to keep it in because so we know it matches `CFVariationColors`

Comment (knagurski) [reply src/modules/75-cf/constants/index.ts:100]: 🤢

**Review by knagurski — APPROVED** [review]
🔥

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**